tools/restatectl/src/commands/metadata/mod.rs

```diff
@@ -39,15 +40,14 @@ pub enum Metadata {
 #[derive(Args, Clone, Debug)]
 #[clap()]
 pub struct MetadataCommonOpts {
-    /// Metadata store server addresses
+    /// Etcd store server addresses
     #[arg(
         short,
-        long = "addresses",
-        default_values = &["http://127.0.0.1:5122"],
-        env = "RESTATE_METADATA_ADDRESSES",
-        value_delimiter = ','
+        long = "etcd",
+        value_delimiter = ',',
+        required_if_eq("remote_service_type", "etcd")
     )]
-    addresses: Vec<String>,
+    etcd: Vec<String>,
 
     /// Metadata store access mode
     #[arg(long, default_value_t)]
```

Comment:
Unrelated to this PR: I am unsure whether we should keep the `MetadataAccessMode::Direct`. Maybe this is something that we can clean up once the metadata accesses go through the Restate server.

Reply:
Yes, definitely